airbyte-integrations/connectors/source-zendesk-support/source_zendesk_support/schemas/organization_fields.json

```diff
@@ -0,0 +1,50 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "title": "Organization_fields Schema",
+    "additionalProperties": true,
+    "type": "object",
+    "properties": {
```

Comment:
Please add the following missing properties: `custom_field_options, relationship_filter, relationship_target_type, system, tag` per [the documentation](https://developer.zendesk.com/api-reference/ticketing/organizations/organization_fields/#json-format).

Also, the indentation needs to be fixed.